test/utils/sesar_api_test.py

```diff
@@ -0,0 +1,87 @@
+
+import inspect
+from pytest import raises
+
+from sample_uploader.utils.sesar_api import (
+    _get_igsn_endpoint,
+    retrieve_sample,
+)
+
+
+def start_test():
+    testname = inspect.stack()[1][3]
+    print('\n*** starting test: ' + testname + ' **')
+
+
+def fail_retrieve_sample(igsn, error_msg, error_type, contains=True):
+
+    with raises(Exception) as context:
+        retrieve_sample(igsn)
+
+    assert type(context.value) == error_type
+
+    if contains:
+        assert error_msg in str(context.value.args[0])
+    else:
+        assert error_msg == str(context.value.args[0])
+
+
+def test__get_igsn_endpoint():
```

Comment:
Nit: This tests seems a bit redundant at the moment, is there a plan to make the igsn endpoint function return a different value depending on context/arguments? Otherwise maybe it'd be better as a constant?

Reply:
yea. I am not sure if we want to work on other endpoints. so currently I make a func to return a very apparent thing.